platformio.ini

```
@@ -11,10 +11,7 @@
 platform = espressif32
 framework = espidf
 
-[env:pico32]
-board = pico32
-monitor_speed = 115200
-
 [env:nodemcu]
 board = nodemcu-32s
-monitor_speed = 115200
\ No newline at end of file
+monitor_speed = 115200
+build_flags = -DCONFIG_WIFI_SSID=\"ESP_AP\" -DCONFIG_WIFI_PASSWORD=\"MYPASS\"
```

```suggestion
build_flags =
  -DCONFIG_WIFI_SSID=\"ESP_AP\"
  -DCONFIG_WIFI_PASSWORD=\"MYPASS\"
```
it's more readable this way. but please check if an empty first line is permitted